ntbs-service/Pages/_HomepageKpiDetails.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@model ntbs_service.Pages.IndexModel
+
+<div>
+    <filtered-homepage-kpi inline-template>
+        <div id="homepage-kpi-details">
+            <div class="flex-container">
+                <label nhs-label-type="Standard" for="code-filter-dropdown" classes="homepage-kpi-filter-label" >
+                    Filter by code
+                </label>
+                <select id="code-filter-dropdown" nhs-select-type="Standard" ref="filterByCode" asp-items="Model.Codes"
+                    v-on:change="updateKpiDetails" classes="homepage-kpi-filter-dropdown">
+                </select>
+            </div>
+            <br/>
+            @foreach (var item in Model.HomepageKpiDetails)
+            {
+                <div id="kpi-@item.Code" v-if="CodeFilter == '@item.Code'">
+                    <nhs-grid-row>
+                        <nhs-grid-column grid-column-width="OneQuarter" classes="homepage-kpi-card">
+                            <div>
+                                <h1>@item.PercentPositive</h1>

```

Comment:
I would be tempted to give it a hidden h1 to identify that this section is for KPIs (screen reader only) and have these as h2

Reply:
Why should we make it h2?

Reply:
Feels a little iffy for 4 things which are semantically of the same section - to each define a top level heading.
It would just give a little more context to screen readers.
Pretty happy to be ignored on it though.

Reply:
In fact, I don't think they should be `hX`s at all - semantically these are not headings. I would be tempted to just make them `div`s and give them appropriate size through css - I'm not sure if a more semantically accurate element exists.